ui/src/testing/default-server-config.ts

```diff
@@ -11,11 +11,17 @@ const defaultAccessModuleConfig: AccessModuleConfig[] = [
     requiredForRTAccess: true,
     requiredForCTAccess: true,
   },
+  {
+    name: AccessModule.RASLINKIDME,
+    expirable: false,
+    requiredForRTAccess: false,
+    requiredForCTAccess: false,
+  },
   {
     name: AccessModule.RASLINKLOGINGOV,
     expirable: false,
-    requiredForRTAccess: true,
-    requiredForCTAccess: true,
+    requiredForRTAccess: false,
+    requiredForCTAccess: false,
```

Comment:
I agree, but I am trying to think how to best change this, and whether we need to do.

As far as I can tell, these two fields are used for calculating days until a module expires (not relevant for identity providers) and whether or not to show the RT and CT badges on the Access modules table on the user admin page. I am wondering what we want to show here, because neither of them are strictly required on their own, but it is an either or situation. Is no badge ok?

I am up for suggestions.

![image](https://github.com/all-of-us/workbench/assets/24514630/9d2cc121-3f88-411e-b620-8c2265143647)

Reply:
Working backward from what we want the UI to do makes sense.

The badges should be updated in some way. Personally, I haven't used the badges to make any decisions so I'd be OK with removing them. However, I've only ever bypassed test users. It's worth checking with the team if that's helpful for real-user bypasses and what they'd prefer in that field.

We could return "days until a module expires" instead if the badges aren't needed.